### DISCUSSION TITLE
Kestrel availability - sad

### DISCUSSION THREAD

**Discussion by esksp** [endless-sky/endless-sky#10744] [Balance, Mechanics, and Interactions]
- Attn: Mild Spoilers ahead

**I stumbled upon the Kestrel quest and it is now available for purchase -- unfortunately too late.**
By now I have access to much better alien ships, so why bother with puny human tech anymore? :-(
A pity, I really like this ship's looks.

I checked the Kestrel file and saw it's unlocked by your combat rating, why not. For the record my current combat rating is 36991, that's over 30k more than the 6k required, but then again I was occupied with Hai/Remnant/Wanderer story lines, so the chance I'd land on _that specific planet_ and get this mission was vanishingly small...

I'm at a loss as to how one could avoid this: **The problem is that after the first big human story line** (Free Worlds for me), **you'll eventually end up leaving human space and start exploring the countless aliens around. After that there is simply no reason to come back to human space** (except moving through), human tech is totally outdated once you've been accepted by the Hai and Remnants (not to mention the Wanderers).
I assume the Kestrel quest was designed in a time when there was less alien stories and players spent more time moving around in human space? 

A solution could be to add some must-do stints in human space, like the "vanishing Sestor fleet" incident, but a) place it where it can trigger the Kestrel quest, and b) put it _early enough_, so that the Kestrel still makes some sense.
The third possibility would be to make the Kestrel better, so it remains a valid choice compared to (for instance) a Shield Beetle. Yes, it's cheaper, but at that point in the game you're not quibbling about a million more or less, and don't forget you'll also need to pay to replace all those tired human outfits with better alien ones.
Don't get me wrong, the Kestrel is by far the best _human_ warship, but after some point in the game _human_ ships don't really cut it anymore... And this is unfortunate.

Now I totally agree humans should be outclassed, but maybe not that much, or at least not completely? Maybe they should retain some specific ship/weapon or combination thereof which can stand the comparison? Of course it would be very hard to get, to avoid getting one too soon, but still remain a not-entirely-stupid choice once you're swimming in Remnant/Wanderer and stolen Korath tech??
Food for thought.

(For me I've solved the problem my way, but that's me.) -grin-

**Comment by Saugia** [comment]
The Kestrel is purely an easter egg - we're not concerned with when it is received. It has a number of times now been given more lose restrictions/earlier obtainment, and we've said that the last time is when we'd last do it. It's not meant to be relevant, it's purely an easter egg call back to EV.

As an aside, in general, the relativity concern applies to human warships. They're not *supposed* to be relevant at all times. The Kestrel is still  one of if not the best human ships, easter egg or not - and it can still be used late game (many people do so). Ship relevance and progression is based on where you are at in the story - not something that needs to be brought with you later into it. 

For example, if you're playing the FW campaign, you're expected to make use of their outfits and ships (mainly the Dreadnaught). If you're later into the Wanderer Campaign, you're expected to be using the Wanderer's ships. Or, rather, that's what its balanced around (with a single or a few ships). You're more than able to get a fleet of lower tier ships though and still do the content.

**Reply by Saugia** [reply]
See: https://github.com/endless-sky/endless-sky/pull/9988 for the most recent discussion on the Kestrel (and previous ones).

**Reply by esksp** [reply]
>  it's purely an easter egg call back to EV.

I see.
I didn't know that.
Still a pity though... :-)

> Ship relevance

Yes, true, but I feel it is a waste to completely abandon human space in the mid/late game. It's a huge block of galaxy... Of course there are lots if interesting things to do elsewhere, but it feels like a waste. There could be things happening, but it is unlikely you'll ever stumble upon them because, well, there is nothing of interest to push you to go back there: Tech is low quality, jobs are not very well paid, hunting pirates is like hunting mosquitoes (and about as profitable), and so on. :-(

> See: #9988 

Thanks!

**Reply by Saugia** [reply]
No worries. We're in dire need of adding a "Frequently asked questions/requests" page and have talked about making one eventually, and the Kestrel rating is definitely one of those things.

**Reply by Saugia** [reply]
It ends up being how the main campaign is written. It's kind of similar in the EV games too. Once you complete the main campaign, the fun in going back is playing with all your new alien technology. In EV, you wouldn't really want to fart around in human ships anymore once you got the alien ships. In EVO, similar if you got Voinian/Crescent ships (especially that Igadzra, which I would say is essentially a Worldship or KIV). In EVN, once you got Polaris ships, everything doesn't matter. It just ends up being a result of finishing the story. 

Since there's so many things to do in human space though, it adds replayability, along with those later unlock missions that you could go for earlier.

**Reply by danaris** [reply]
It's also worth noting that once you have unlocked the Kestrel for the first time on a given install, subsequent playthroughs have a _much_ lower threshold for getting the unlock mission.

**Reply by esksp** [reply]
> It's kind of similar in the EV games too

IMHO, EV should just inspire you, not limit you.
I've never played EV, so for me (and I'm sure a majority of current ES players) ES is **a game in its own right, not just a homage to EV**.
For the majority of current ES players the Kestrel is part of ES's lineups of ships, so it should (IMHO) be treated as one, even if it started as something else.

If you want to make it a true Easter egg, make it an author ship no player can obtain. -shrug-

**Reply by danaris** [reply]
If the Kestrel is part of ES's lineup of ships, it's still a human ship, and should fall within human technology bounds.

**Reply by esksp** [reply]
> once you have unlocked the Kestrel for the first time on a given install, subsequent playthroughs have a _much_ lower threshold for getting the unlock mission.

That's the (IMHO pointless) Easter egg perspective: Either the ship is overpowered at CR 2000, or it is just fine. There is no "first" and "second" time...

> it's still a human ship, and should fall within human technology bounds

Sure.
I just question the opportunity to get it. It happens that in the usual flow of the game (as far as I have seen it), it is rather unlikely to happen if you don't know about it. Which I find regrettable, that's all.

Now the fact I find that human tech shouldn't be totally useless beyond the beginning of the game is a different, separate point, which I illustrated with this otherwise fine ship.

**Reply by Saugia** [reply]
> I've never played EV, so for me (and I'm sure a majority of current ES players) ES is a game in its own right, not just a homage to EV.
For the majority of current ES players the Kestrel is part of ES's lineups of ships, so it should (IMHO) be treated as one, even if it started as something else.

It was moreso a comparison; I had thought you came from EV from reading somewhere, so I am mistaken. I was using EV as an example, but we actually run pretty strongly as "ES is its own game vs EV" actually, if you read around Github. In this case, it's what the original Dev of ES intended with the Kestrel, human ships and story progression, and it's what we current developers continue to believe is how it should be. It's okay to come to disagreements sometimes.

Ultimately, the Kestrel has received many PRs through the last several years to obtain it sooner, and we feel as though it does not need any more. Not every ship in the game is supposed to be relevant (there are other Easter egg ships too that aren't amazing when you get them. If we made every ship more relevant, it would close the gaps and make a much more muddied balance field.) We appreciate your input regardless.

**Reply by esksp** [reply]
> I had thought you came from EV from reading somewhere

No.  :-)
For the record, I accidentally stumbled upon ES on GOG while looking for some SciFi game to waste some time on for a week or two... Oh well, that went much further than I expected...  :-D

> It's okay to come to disagreements sometimes.

Of course.

> Not every ship in the game is supposed to be relevant

It's not about relevance, it's about consistency: You've heard of "[Chekhov's gun](https://en.wikipedia.org/wiki/Chekhov%27s_gun)"? If in a piece of fiction you place something prominently somewhere, the spectator/player will expect it to have some meaning or purpose in the film/game.

For this reason Easter eggs (which are just private jokes) should remain invisible, i.e. if that Easter egg is a person, you don't introduce him at length just to drop him afterwards, no, you just have him passing somewhere in the back of the picture so that people who understand the hint might notice him, but so he still remains invisible for the others. Remember Alfred Hitchcock's famous Easter egg appearances: He just walked by in the background once, he wasn't introduced as some secondary-but-ultimately-useless role. That's why I said that Easter egg ships (I understand there might be others) should be limited to unobtainable "Author" ships. People who get the hint notice, for the others it's just another of those strange "author" ships.  
Just my 2 cents worth... I'm not saying or expecting you should do it.

**Reply by TheGiraffe3** [reply]
Note that the Kestrel is an author ship as well, flown by Michael Zahniser, ES' original creator.

**Reply by esksp** [reply]
> Note that the Kestrel is an author ship as well

It would had surprised me if it didn't... :-)
My point still stands: The "author" ship version would had been enough for an "Easter egg", no need to make a huge "Easter omelet" out of it... :-D
It's still a cool ship, and as a player-usable ship it deserves a better chance to shine, IMHO.

**Comment by Hecter94** [comment]
I like the Kestrel as it is currently. 

If you do a lot of early exploration, take your time working through missions, and happen to stumble across it early, it's a very powerful tool that's a really fun reward. 
If you don't do much early exploring and only find it late, it's not as powerful as a reward, but it's still very cool to have a ship whose specialty you can customize through dialogue, especially if you go for the weapons variant. Its massive number of gunports gives it a pretty strong niche, which is second only to the Chigiriki, which can be a pretty late-game find.
And, for those who've come from EV, I imagine even just seeing it again is neat, regardless of its power.

Once you've found it, it presents a fun question: "How early could I get it?" It incentivizes you to try a second playthrough to see if you can get that powerful tool as early as possible and even rewards you by making it _easier_ to get on a second playthrough, so you're doubly rewarded if you have that thought and decide to try it out. 
Many people praise ES for its open-ended nature and the ability to go through the same content in a different context if you run through the game in a different order. I see the Kestrel as being a part of that, something that acts as a super cool reward if you happen to find it early and a reason to try replaying if you don't find it early. 

Ultimately, if it were to become something that every player is "expected" to find early, it would stop being special and fun and just become another link in a linear progression chain, which I feel would be an overall loss.

**Reply by esksp** [reply]
> happen to stumble across it early

That would only happen if you have a very specific play style (combat-oriented) and a lot of luck. I started my career doing mostly transport jobs (from the job center), and avoiding combat by turning my ship into a hot rod. My combat rating only started to rise during the FW story line, where I couldn't avoid combat... 

> Its massive number of gunports gives it a pretty strong niche

True, but defense-wise (shields/hull) it is subpar compared to the ships you have access to at that point.
I'm not very agile in the fingers anymore, so I appreciate my shields and hull giving me some additional seconds of reaction time...
Of course you could go for the "improved shields" version (which I did), but the result would still have inferior protection compared to a Shield Beetle for instance.

> it presents a fun question: "How early could I get it?"

Really? Not for me.
On _my_ second career my only thought was "how early can I get to Hai space, and then to Remnant space? (Answer: "Right from the start"...) And once you have access to Hai and Remnant tech, human tech becomes finally outdated (thus my ideas above about making that a little less final).

> Many people praise ES for its open-ended nature

Sure, but that is not in question here. My initial post was _about the risk of finding the Kestrel way too late, because of how the story line naturally flows_.


> Ultimately, if it were to become something that every player is "expected" to find early

I didn't ask to find it early! If only because it would make all other human warships useless. You can _not_ make it appear early or easily.
I repeat, I just said **currently there is a risk for the unsuspecting new player to find it at a point it has become totally pointless**, that's all.  :-)

**Reply by Hecter94** [reply]
> currently there is a risk for the unsuspecting new player to find it at a point it has become totally pointless

Ultimately, it comes down to a very simple question, 
"Are we okay with players potentially 'missing' content?"
_In this case, I'm also considering "finding content out of the intended order" to mean "missed"._

I would strongly argue, "Yes, we should be okay with the idea that some things will be skippable or missable."

I believe that is what it means for a game to be considered "open-ended." Different people will have different experiences with it, including missing certain elements or experiencing things in a different order.
Some individuals will play the game in one way and experience content accordingly. Others may choose a different play style and encounter different content.

The Kestrel is an example of a cool reward for players who play the game in a certain way and an option for replaying if they missed it the first time. 
The fact that it can be potentially missed is, in my opinion, an example of the game's open-ended nature working exactly as designed. The possibility of missing it makes finding it special for those who do.

On a somewhat related note, and I apologize for the dated reference. Still, I recently replayed [Armored Core](https://en.wikipedia.org/wiki/Armored_Core_(video_game)), and there's a specific hidden weapon in that game that I think is quite relevant to the current discussion.

In one of the very first missions in the game, potentially _the_ first mission you will ever do, hidden directly behind your starting location, is one of if not the hardest hitting [weapon](https://armoredcore.fandom.com/wiki/WR-L24) in the entire game.
Most players aren't going to find that. Unlike Endless Sky, where you may eventually find the Kestrel even if you missed it originally, with this weapon, if you miss it the first time, you won't be able to get it later, so you just permanently miss out on one of the hardest hitting weapons in the entire game. 

This is great; the people who do find it are given an excellent reward, and because they had to look for it specifically and are rewarded for their effort, they have a great experience. 
Those who missed it have a reason to try replaying the game, pick it up next time, and see how finding that weapon changes their experience with the game. 

Considering the series has continued this trend in some form or another, even to its latest entry, Armored Core 6, I would say it's clearly working out well for them.

The Kestrel fulfills that same purpose; the risk of finding it "too late" or even missing it entirely is what makes it special.

**Reply by esksp** [reply]
> I believe that is what it means for a game to be considered "open-ended."

I have a different definition, but that's no problem. :-)

BTW many games have hidden "start helpers". I remember for instance the first S.T.A.L.K.E.R. having a hidden armor in one of the houses where your started. You had to manage to climb on a roof, jump to the next house, and do some acrobatics to eventually find an armor hidden in a hole, which made the start much easier. Your chances of finding it by pure chance were negligible, but it was the favorite hint on Internet...
Those are IMHO different things, they are there to help you _start_, and are available right from the very beginning.
Transferring this to ES it would be that some improbable combination of choices gives you a heavy warship for free before even taking off from your start planet.

Anyway, the gist of your whole post is that you consider the status quo to be exactly like you want it, so let's just agree to disagree, you won't convince me, at least not that way... :-D
No hard feelings.
